src/time.cr

```diff
@@ -253,8 +253,17 @@ struct Time
   DAYS_PER_4_YEARS = 365*4 + 1
 
   # :nodoc:
+  # TODO: remove after 0.27.0, and replace the value of UNIX_EPOCH by "Time.utc 1970, 1, 1"
```

Comment:
Why can't this constant be removed right away? It's only used internally and since it is no longer used, what's the purpose of keeping it until the following release?
Plus, the value of `UNIX_EPOCH` should be `utc(SECONDS_PER_DAY.to_i64 * (1969*365 + 1969/4 - 1969/100 + 1969/400))`.